### ISSUE TITLE
Error in fresh pip install: daal=2021.4.0

### ISSUE THREAD

**Issue by viochemist** [gbouvignies/ChemEx#61]
pip install returned an error at the end saying:

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
daal4py 2021.5.0 requires daal==2021.4.0, which is not installed.

This is not a package I'm familiar with, so I don't know if it's critical. I was able to install daal==2021.4.0 subsequently.

**Comment by viochemist** [comment]
So, I just "git clone chemex", go in the directory, and then do "pip install ." This has been the most reliable approach for me recently. I'm amenable to other means though.

**Comment by viochemist** [comment]
Update: I'm working with "poetry" and, again with a freshly installed minconda environment, I get:
```

(base) ahansen@3096W52-AHANSEN:~/Programs/ChemEx$ poetry install
The currently activated Python version 3.9.7 is not supported by the project (=3.9).
Trying to find and use a compatible version. 

  NoCompatiblePythonVersionFound

  Poetry was unable to find a compatible version. If you have one, you can explicitly use it via the "env use" command.

  at ~/Programs/miniconda/lib/python3.9/site-packages/poetry/utils/env.py:736 in create_venv
       732│                     python_minor = ".".join(python_patch.split(".")[:2])
       733│                     break
       734│
       735│             if not executable:
    →  736│                 raise NoCompatiblePythonVersionFound(
       737│                     self._poetry.package.python_versions
       738│                 )
       739│
       740│         if root_venv:


```

I can create a new environment without problem, I just wanted to check first if ChemEx really needs 3.9 exactly or 3.9.x?

**Comment by gbouvignies** [comment]
Hi Alex,

Sorry for the trouble. I had to change the python version in the `pyproject.toml` file because `scipy` complained about it. I thought that `python = "=3.9"` was selecting `3.9.x`, but it does not seem to be the case on your side.

One way to get around it cleanly is to change this line to:
```toml
python = ">=3.9,<3.11"
```

Let me know if that works for you.

**Comment by gbouvignies** [comment]
Fixed in version 2022.0.0.
